Examples/Scripts/Python/itk.py

```diff
@@ -258,24 +266,33 @@ def buildITkGeometry(geo_dir: Path, material: bool = True):
                         [535.009, 559.101],
                         [559.101, 574.194],
                     ],
-                    "EC2": [[575.594, 606.402], [606.402, 637.209]],
+                    "EC2": [
+                        [575.594, 606.402],
+                        [606.402, 637.209]
+                    ],
                     "EC3": [
                         [638.609, 670.832],
                         [670.832, 697.055],
                         [697.055, 723.278],
                         [723.278, 755.501],
                     ],
-                    "EC4": [[756.901, 811.482], [811.482, 866.062]],
-                    "EC5": [[867.462, 907.623], [907.623, 967.785]],
+                    "EC4": [
+                        [756.901, 811.482],
+                        [811.482, 866.062]
+                    ],
+                    "EC5": [
+                        [867.462, 907.623],
+                        [907.623, 967.785]
+                    ],
                 },
             ),
             Volume(
                 name="HGTD",
-                layers=LayerTriplet(positive=True, central=False, negative=True),
-                subVolumeName=LayerTriplet("HGTD::HGTD"),
                 binToleranceR=(15 * u.mm, 15 * u.mm),
-                binToleranceZ=(15 * u.mm, 15 * u.mm),
+                binToleranceZ=(5 * u.mm, 5 * u.mm),
                 binTolerancePhi=(0.025 * u.mm, 0.025 * u.mm),
```

Comment:
I noticed also that the HGTD `binTolerancePhi` is 0.025 in Python, but `geo-tgeo-sfbin-phi-tolerance` is 0.25 in JSON. If I change the Python to 0.25, I get identical results from the two methods.
Do you know which is better?

Reply:
I have changed also this parameter to use the same we have in the JSON file.
Not sure where this number come from though.